dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/model/Contentlet.java

```diff
@@ -197,7 +197,13 @@ public void setInode(String inode) {
      * @return
      */
     public long getLanguageId() {
-    	return (Long)map.get(LANGUAGEID_KEY);
+    	try {
```

Comment:
You can do something like ConversionUtils.toLong (map.get(LANGUAGEID_KEY), 0);

Reply:
in fact, it already exists